lib/task_helpers/imports/custom_buttons.rb

```diff
@@ -72,11 +87,30 @@ def check_user(new_obj)
           existing_user = User.find_by(:name => new_obj[:userid])
           new_obj.update(:userid => existing_user.nil? ? "admin" : existing_user)
         end
+
+        private
+
+        def populate_item(item, obj, class_name)
+          item.tap do |i|
+            add_children(obj, i)
+            add_associations(obj, i)
+            try("#{class_name}_post", i)
+          end
+        end
+
+        def populate_action(item, assoc)
```

Comment:
These two methods added here are not necessary. Please move the code into `add_associations`.